src/shared/lib/StringUtils.ts

```diff
@@ -15,4 +15,40 @@ export function longestCommonStartingSubstring(str1:string, str2:string) {
     }
 
     return str1.substring(0, i);
+}
+
+export function formatDecimal(x:number, numDigitsAfterDecimal:number) {
```

Comment:
I think javascript's toFixed method does this. https://www.w3schools.com/jsref/jsref_tofixed.asp